### PR TITLE
Change debug message on autodiscover to report actual configurations

### PR DESCRIPTION
**Pull request by vjsamuel**

### CODE REVIEW

File: libbeat/autodiscover/autodiscover.go [hunk 1]
```diff
@@ -92,6 +93,8 @@ func NewAutodiscover(name string, pipeline beat.Pipeline, adapter Adapter, confi
 		runners:         cfgfile.NewRunnerList("autodiscover", adapter, pipeline),
 		providers:       providers,
 		meta:            meta.NewMap(),
+
```
Comment (ruflin) [inline libbeat/autodiscover/autodiscover.go]: Nit: remove newline?

File: libbeat/autodiscover/registry.go [hunk 2]
```diff
@@ -40,5 +46,7 @@ func NewRegistry() *registry {
 		providers: make(map[string]ProviderBuilder, 0),
 		builders:  make(map[string]BuilderConstructor, 0),
 		appenders: make(map[string]AppenderBuilder, 0),
+
```
Comment (ruflin) [inline libbeat/autodiscover/registry.go]: Nit: Seems like this newline is intentional?

**Comment by ruflin** [comment]
jenkins, test this

File: libbeat/autodiscover/autodiscover.go (hunk 1 above)
Comment (vjsamuel) [reply libbeat/autodiscover/autodiscover.go]: done

File: libbeat/autodiscover/registry.go (hunk 2 above)
Comment (vjsamuel) [reply libbeat/autodiscover/registry.go]: done

**Comment by exekias** [comment]
jenkins test this

**Comment by ruflin** [comment]
Looks like this needs a `make fmt`

**Comment by vjsamuel** [comment]
@ruflin did a rebase and `make fmt`

**Comment by ruflin** [comment]
jenkins, test this

**Comment by ruflin** [comment]
@exekias I see you added `needs_backport`. Do you want this in 7.0? I think 7.1 is fine.

**Pull request merged**